lua/nvim-tree.lua

```diff
@@ -370,8 +377,16 @@ local function setup_autocommands(opts)
   end
 
   if not opts.actions.open_file.quit_on_open then
-    create_nvim_tree_autocmd("BufWipeout", { pattern = "NvimTree_*", callback = view._prevent_buffer_override })
+    create_nvim_tree_autocmd("BufWipeout", {
+      pattern = "NvimTree_*",
+      callback = function()
+        if api.nvim_buf_get_option(0, "filetype") == "NvimTree" then
```

Comment:
vim.bo.filetype is more luaish

Reply:
Today I Learned

Reply:
`vim.bo.filetype` is also a shorthand for `vim.bo[0].filetype`.

Reply:
Fantastic. I might merge these checks into `is_nvim_tree_buf`.